meeting_minutes/230119_SARIF_TC_72.md

```diff
@@ -0,0 +1,158 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company           | Role(s)                         |
+|:-----------|:----------|:------------------|:--------------------------------|
+| Adar       | Weidman   | JFrog             | Member.                         |
+| Aditya     | Sharad    | Microsoft         | Voting Member                   |
+| David      | Keaton    | Individual        | Chair                           |
+| David      | Malcolm   | Red Hat           | Guest                           |
+| Gerald     | Sullivan  | Micro Focus       | Voting Member                   |
+| Mary       | Martin    | Microsoft         | Voting Member                   |
+| Michael    | Fanning   | Microsoft         | Voting member                   |
+| Nathan     | Baird     | Microsoft         | Voting Member                   |
+| Paul       | Anderson  | Grammatech, Inc.  | Member                          |
```

Comment:
Paul is a voting member.  There was a time last year when he lost voting rights, but he regained them later.  I think this is just a cut-and-paste error that I had not caught until now.